plugins/fingerquoter_test.go

```diff
@@ -165,38 +168,62 @@ func TestQuotingOfSingleLongWord(t *testing.T) {
 	f, err := plugins.NewFingerQuoter(pc)
 	assert.Nil(t, err)
 
-	h := f.HearActions[0]
+	assertplugin := assertplugin.New("bot")
 
-	assert.Equal(t, "\"belong\"", h.Answer(&slackscot.IncomingMessage{Msg: slack.Msg{Text: "Do I belong or not?", Timestamp: "1546833210.036900"}}).Text)
+	assertplugin.AnswersAndReacts(t, &f.Plugin, &slack.Msg{Text: "Do I belong or not?", Timestamp: "1546833210.036900"}, func(t *testing.T, answers []*slackscot.Answer, emojis []string) bool {
+		return assert.Len(t, answers, 1) && assertanswer.HasText(t, answers[0], "\"belong\"")
+	})
+}
+
+func TestNotQuotingPartsOfURLs(t *testing.T) {
+	pc := viper.New()
+	pc.Set("frequency", 1)
+
+	f, err := plugins.NewFingerQuoter(pc)
+	assert.Nil(t, err)
+
+	assertplugin := assertplugin.New("bot")
+
+	assertplugin.AnswersAndReacts(t, &f.Plugin, &slack.Msg{Text: "http://google.com/query?bigfoot=friend", Timestamp: "1546833310.036900"}, func(t *testing.T, answers []*slackscot.Answer, emojis []string) bool {
```

Comment:
👎 I realized after posting that there's one edge-case the `regexp` doesn't handle well and that's if the first word is 5 characters or longer but part of a url (or something which isn't a sentence-like string).